src/contentTypeConfig.js

```diff
@@ -14,7 +14,7 @@ export const DEFAULT_FIELDS = [
   "'id': railcontent_id",
   'railcontent_id',
   artistOrInstructorName(),
-  'artist',
+  "'artist': select(artist != null => { 'name': artist->name, 'thumbnail': artist->thumbnail_url.asset->url}, null)",
```

Comment:
I think you can make this simplier. If artist is null the whole object is null.
```suggestion
  "'artist': artist->{ 'name': artist->name, 'thumbnail': artist->thumbnail_url.asset->url},",
```